fs/operations/operations.go

```diff
@@ -1178,14 +1176,7 @@ func Cat(ctx context.Context, f fs.Fs, w io.Writer, offset, count int64) error {
 				size = count
 			}
 		}
-		in = accounting.NewAccountSizeName(in, size, o.Remote()).WithBuffer() // account and buffer the transfer
-		defer func() {
```

Comment:
Where does `in` get closed?

Maybe in `accounting.Stats.DoneTransferring`?  That seems a bit too magic for me.